_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,122 @@
+
+```bash
+gruntwork aws revoke \
+            --account "dev" \
+            --account "stage" \
+            --account "prod" \
+            --account "security" \
+            --account "shared" \
+            --account "logs"
+```
+
+To revoke Gruntwork's access from the "current" account—the one you are authenticated to in the terminal—use the name
+`__current__` (i.e., `--account "__current__"`).
+
+
+**Important**: Never use the `AdministratorAccess` AWS managed policy with any users, groups, or roles. It gives full access to all resources. Instead, to give access to administrators, use a policy that allows full IAM privileges (e.g. `iam:*`) on all resources. Our Landing Zone solution created a `iam-admin` group in the Security account. [More information about the existing groups in the `iam-groups` module.](https://github.com/gruntwork-io/terraform-aws-security/tree/main/modules/iam-groups)
+
+The steps below should be performed on each deployed account.
+
+## 2. Enable MFA for the root user
+
+Securing the "root" user, or the first user that is created when you set up an AWS account, is one of the
+first actions you should take in any new account. It is highly recommended that you avoid using this root user for everyday tasks. Unfortunately, there is no API or automation available for configuring an MFA device for the
+root user. Follow the manual steps outlined in the
+[AWS docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_mfa). Configuring a virtual MFA device will achieve recommendation 1.5. You can also refer to the [production-grade AWS account structure guide.](https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/)
+
+Configure a hardware MFA device, as suggested by recommendation 1.6. We suggest using a
+[Yubikey](https://www.yubico.com/) due to its strong security characteristics and multitude of form
+factors. Refer to
+[the documentation for more information on using a hardware device with the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_physical.html#enable-hw-mfa-for-root).
+
+## 3. Answer security questions and complete contact details
+
+When setting up a new account, AWS asks for contact information and security questions. Unfortunately, there is no API or automation available for this functionality. In the AWS console, visit the [Account settings](https://console.aws.amazon.com/billing/home?#/account) page and complete the _Alternate Contacts_ and _Configure Security Challenge Questions_ questions.
+
+
+## 4. Delete default VPCs and rules from default security groups
```

Comment:
This isn't necessary for non-CIS users right? If so, we should specify that.

Reply:
This whole page is for CIS users, so I will leave as it is.